recipes/lsstdesc.coord/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "LSSTDESC.Coord" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "fca209035123683cc666aa20d01ea5f67b31f370f2f5195af8a7e8db8bda484f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.13
+    - future >=0.15
+    - cffi >=1.11.0
+
+  run:
+    - python
+    - numpy >=1.13
+    - future >=0.15
+    - cffi >=1.11.0
+
+test:
+  imports:
+    - coord
+
+about:
+  home: https://github.com/LSSTDESC/Coord
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Could you please also open a PR in the project to ask them to package the license as well? just adding the license in the ``MANIFEST`` should be fine

Reply:
They may already. Let me check.

Reply:
Done. https://github.com/LSSTDESC/Coord/pull/9

Reply:
Not necessary for now, but for the next versions that would be good! :)